src/TestHelper/src/System/Diagnostics/RemoteExecutorTestBase.netcore.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace System.Diagnostics
+{
+    /// <summary>Base class used for all tests that need to spawn a remote process.</summary>
+    public abstract partial class RemoteExecutorTestBase : FileCleanupTestBase
+    {
+        protected static readonly string HostRunnerExecutableName = "dotnet";
```

Comment:
HostRunnerName is used in the S.D.Process tests. but I'll look at the ones not used outside here and will remove them.